sklearn/decomposition/online_lda.py

```diff
@@ -11,11 +11,15 @@
 # Author: Chyi-Kwei Yau
 # Author: Matthew D. Hoffman (original onlineldavb implementation)
 
+import warnings
+
 import numpy as np
 import scipy.sparse as sp
-from scipy.misc import logsumexp
 from scipy.special import gammaln
-import warnings
+try:  # SciPy >= 0.19
+    from scipy.special import logsumexp
+except ImportError:
+    from scipy.misc import logsumexp
```

Comment:
i would uses fixes.py for this to avoid the try except in all our examples

you would write:

from sklearn.utils import logsumexp